src/Pages/GradeSelect/FlightListCard/index.js

```diff
@@ -0,0 +1,262 @@
+    border-radius: 14px;
+    padding: 2px 10px;
+    color: #555;
+    font-size: 14px;
+    line-height: 1.58;
+    cursor: pointer;
+    background: 0 0;
+  `;
+
+  const FlightFrom = styled.div`
+    flex: 1;
+    font-size: 12px;
+    text-align: left;
+  `;
+  const Timeline = styled.span`
+    display: block;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 1.5;
+  `;
+
+  const AirCode = styled.span`
+    display: inline-block;
+    color: #767676;
+    vertical-align: top;
+    white-space: nowrap;
+    font-size: 14px;
+    line-height: 1.5;
+  `;
+
+  const FlightTo = styled.div`
+    flex: 1;
+    font-size: 12px;
+    text-align: right;
+  `;
+
+  const SeatSelection = styled.div`
+    width: calc(1281px - 472px);
+    margin-left: -1px;
+  `;
+
+  const SeatSelectionWrap = styled.div`
+    display: flex;
+    overflow: visible;
+    height: 100%;
+    border: 1px solid #d9dbe1;
+    box-shadow: none;
+  `;
+
+  const Standard = styled.div`
+    flex: 1 1 30%;
+    position: relative;
+    border-right: 1px solid #d9dbe1;
+    text-align: center;
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: -1px;
+      left: -1px;
+      right: 0;
+      border-radius: 2px;
+      border-top: 2px solid #00256c;
+    }
+  `;
+
+  const FareWrap = styled.div`
+    padding: 48px 10px 20px;
+  `;
+
+  const GradeTitle = styled.span`
+    display: inline-block;
+    width: 100%;
+    margin: 0 2px;
+    color: #000;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 1.5;
+  `;
+
+  const GradeFare = styled.span`
+    font-size: 16px;
+    line-height: 1.8;
+  `;
+
+  const Fare = styled.span`
+    font-size: 20px;
+    font-weight: bold;
+    margin-right: 4px;
+  `;
+
+  const SeatStock = styled.span`
+    position: relative;
+    width: 100%;
+    display: inline-block;
+    color: #de001b;
+    margin: 0;
+    font-size: 14px;
+    line-height: 1.6;
+  `;
+
+  const Economy = styled.div`
+    flex: 1 1 30%;
+    position: relative;
+    border-right: 1px solid #d9dbe1;
+    text-align: center;
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: -1px;
+      left: -1px;
+      right: 0;
+      border-radius: 2px;
+      border-top: 2px solid #00256c;
+    }
+  `;
+
+  const FirstClass = styled.div`
+    flex: 1 1 30%;
+    position: relative;
+    text-align: center;
+    background-color: #f8fbff;
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: -1px;
+      left: -1px;
+      right: 0;
+      border-radius: 2px;
+      border-top: 2px solid #2068d1;
+    }
+  `;
+
+  return (
+    <>
+      <FlightListCard>
+        <FlightInfo>
+          <FlightInfoWrap>
+            <FlightTime>
+              <FlightFrom>
+                <Timeline>10:00</Timeline>
+                <AirCode>ICN</AirCode>
+              </FlightFrom>
+              <FlightTo>
```

Comment:
```suggestion
              <FlightTime type="from">
                <Timeline>10:00</Timeline>
                <AirCode>ICN</AirCode>
              </FlightTime>
              <FlightTime type="to">
```
from과 to의 스타일이 다르지 않다면 조금더 일반적인 네이밍으로 하나만 써주는 건 어떨까요?
달라지는 부분이 있더라도 props를 넘겨줄 수 있을거 같아요